installer/Migration/V5_1_0/translation/de.yaml

```diff
@@ -84,3 +84,16 @@ translations:
     - { target: 'Leistungsbeurteiler', unitId: employee_performance_trackers, group: performance }
     - { target: 'hinzugefügt am', unitId: added_date, group: performance }
```

Comment:
@RajithaKumara We don't need to continue this time consuming part for manual generation of translation ymls right?

Reply:
@SharukaNP in your current open PR, delete these yaml files by making a backup